### PR TITLE
DTSPB-2632: Blank letter template

### PR DESCRIPTION
**Pull request by nigeldunne**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPB-2632

### Change description ###

Blank letter tempalte

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by itoulou** [comment]
Code looks good but can't see any FT's testing content on new template

File: src/main/java/uk/gov/hmcts/probate/transformer/assembly/AssembleLetterTransformer.java [hunk 1]
```diff
@@ -290,14 +290,18 @@ private Map<ParagraphCode, BiFunction<ParagraphCode, CaseData, List<ParagraphDet
...
         CaseData caseData = caseDetails.getData();
-        Categories categories = caseData.getCategories();
-        List<CollectionMember<ParagraphDetail>> paragraphDetails = new ArrayList<>();
-        addParagraphsForUsedFields(paragraphDetails, Caseworker.getParagraphFields(), caseData);
-        addAllCategoryParagraphs(paragraphDetails, categories.getAllSelectedCategories(), caseData);
-        addParagraphsForUsedFields(paragraphDetails, FreeText.getParagraphFields(), caseData);
-
-        responseCaseDataBuilder.categories(categories);
-        responseCaseDataBuilder.paragraphDetails(paragraphDetails);
+        if ("blank".equals(caseData.getLetterType())) {
```
Comment (sanjayparekh-hmcts) [inline src/main/java/uk/gov/hmcts/probate/transformer/assembly/AssembleLetterTransformer.java]: Set these letter types as constants / enums? may also need a commons change

File: src/main/java/uk/gov/hmcts/probate/service/DocumentGeneratorService.java [hunk 2]
```diff
@@ -200,8 +200,13 @@ public Document generateLetter(CallbackRequest callbackRequest, boolean forFinal
             placeholders.putAll(mappedImages);
         }
 
-        return pdfManagementService.generateDocmosisDocumentAndUpload(placeholders,
-            DocumentType.ASSEMBLED_LETTER);
+        if ("blank".equals(placeholders.get("letterType"))) {
```
Comment (sanjayparekh-hmcts) [inline src/main/java/uk/gov/hmcts/probate/service/DocumentGeneratorService.java]: see below

**Review by sanjayparekh-hmcts — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/probate/transformer/assembly/AssembleLetterTransformer.java (hunk 1 above)
Comment (nigeldunne) [reply src/main/java/uk/gov/hmcts/probate/transformer/assembly/AssembleLetterTransformer.java]: Made constants, no commons change needed - transient data

File: src/main/java/uk/gov/hmcts/probate/service/DocumentGeneratorService.java (hunk 2 above)
Comment (nigeldunne) [reply src/main/java/uk/gov/hmcts/probate/service/DocumentGeneratorService.java]: Made constant

**Review by sanjayparekh-hmcts — APPROVED** [review]

**Pull request closed without merging**